src/camera.js

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA
+ */
+
+/** @typedef {import('./libapi.mjs.d.ts').Context} Context */
+
+// To avoid errors for users who use SSR or Hybrid rendering (e.g. Nuxt.js), we need to check if we're in the browser.
+let initModule;
+
+async function initializeModule() {
+  if (typeof window !== "undefined") {
```

Comment:
What will happen if you don't do manual check? I assume it'd fail anyway? I'd rather not use manual condition, especially since we might still add Node.js support via [node-usb](https://www.npmjs.com/package/usb) in the future (I tried it once, it was a bit buggy back then, but might be better today).

Reply:
I just tested and adding the ENVIRONMENT flags during build stage removes all the server-side code so this is no longer needed.